### DISCUSSION TITLE
PREPROCESSING FILE PROBLEM

### DISCUSSION THREAD

**Discussion by ljeft** [NanoComp/meep#2711] [Q&A]
HELLO
I just started to learn meep, I am trying to import `.stp` antenna model for simulation, but I only saw the tutorial of importing GDSII in the document, how to import`. stp` model is still a problem

**Comment by stevengj** [comment]
Meep doesn't implement a feature for STP files.  You will have to find something to convert it to GDSII, or re-implement the structure yourself.

**Comment by stevengj** [comment]
(If you mean STL, not STP, then there is https://www.artwork.com/gdsii/stl2gds/index.htm and probably other tools)
